invenio_rdm_records/oaiserver/services/services.py

```diff
@@ -159,6 +169,8 @@ def update(self, identity, id_, data, uow=None):
         """Update an OAI set."""
         self.require_permission(identity, "update")
         oai_set, errors = self._get_one(id=id_)
+        if oai_set.system_created:
```

Comment:
add permission to allow super users?
It depends who can access backoffice...

Reply:
Not sure about this, if there benefits, I am ok with it, however I link my previous answer as it's related, feel free to reach out IRL if you want to quickly discuss!